amical/oifits.py

```diff
@@ -1013,6 +1013,146 @@ def _plot_CP(ax3, l_dic, dic_color, conv_cp, diffWl=False):
     return max_f_cp
 
 
+def _plot_UV_ifu(ax1, fig, l_dic):
+    l_bmax = []
+    all_U, all_V, all_wl = [], [], []
+
+    for dic in l_dic:
+        tmp = _apply_flag(dic)
+        U = tmp.U
+        V = tmp.V
+        wl = tmp.wl
+        l_bmax.append(tmp.bmax)
+        all_U.append(U)
+        all_V.append(V)
+        all_wl.append([wl] * len(U))
+    all_U = np.array(all_U)
+    all_V = np.array(all_V)
+    all_wl = np.array(all_wl) * 1e6
```

Comment:
this is a very inefficient way to construct arrays with deterministic sizes. I know performance isn't a common concern in AMICAL but this stands out to me: we can make it much quicker.